### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by stefansaraev**
as title says

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -184,13 +175,6 @@ lib/cpluff/stamp-h1
 # /lib/ffmpeg/
 /lib/ffmpeg/
```
Comment (wsnipex) [inline .gitignore]: those too please

**Comment by wsnipex** [comment]
nice!
can you remove the mpeg2 stuff in configure.ac as well?

**Comment by stefansaraev** [comment]
fernet already removed it ;)

**Comment by wsnipex** [comment]
ah, sorry for the noise. Didn't look at current master.

**Comment by stefansaraev** [comment]
jenkins build this please

**Pull request merged**